en/docs/administer/logging-and-monitoring/logging/setting-up-logging.md

```diff
@@ -66,6 +66,29 @@ Please follow below steps to enable logs for a given service component available
     loggers = org-wso2-carbon-user-core, trace-messages, org-apache-coyote,com-hazelcast
     ```
 
+###Enable Logs for a Component via UI
+
+!!! note
+    Enabling logs via UI is only available after the WUM update with the timestamp "1600491803101"
+
+Please follow below steps to enable logs for a given service component available in WSO2 API Manager.
+
+1.  Navigate to the carbon console [https://localhost:9443/carbon](https://localhost:9443/carbon) and sign in with **`admin/admin`** as the credentials.
+Go to "Configure" -> "Logging"
+                                                 
+    ![Logger Carbon UI](../../../assets/img/administer/logging.png)
```

Comment:
please use {{base_path}}

Reply:
The local setup did not show the images when the basePath was provided. Could be an issue with local setup. But now I modified using the basepath.